util/util.go

```diff
@@ -29,6 +32,24 @@ func GenerateUUID() string {
 	return uuid.New().String()
 }
 
+
+func Transport(pURL string) *http.Transport {
```

Comment:
It would be better if we name it something like `TransportProxy` or `ProxyTransport`.

Reply:
ok